Tutorials (#547)
replicatedhq/kots.io: content/vendor/guides/vendor-portal/team/security.md

# Client Security
Security is a core principal here at Replicated.  You can help increase the
security of your team's account by enforcing [strong passwords](#strong-passwords)

> and enabling [Two-Factor Authentication](#two-factor-authentication)

Reviewer: ```suggestion
and enabling [Two-Factor Authentication](#two-factor-authentication).
```

# In this Tutorial

> Learn how to enable strong passwords and enable Two-Factor Authentication

Reviewer: ```suggestion
Learn how to enable strong passwords and enable Two-Factor Authentication.
```

# Prerequisites
* Logged into a [vendor account](todo) as a member with an `admin` policy

# Instructions

## Enabling security measures

> 1. Log into the [Vendor Portal](https://vendor.replicated.com), if prompted

Reviewer: ```suggestion
1. Log into the [vendor portal](https://vendor.replicated.com).
```

> 1. Navigate to the Team menu at the top right of the Vendor Portal

Reviewer: ```suggestion
1. Navigate to the **Team** menu.
```

    ![Team](img/team-top-menu-button.png)

> 1. Click on the **Security** menu on the left

Reviewer: ```suggestion
1. Click the **Security** menu.
```

    ![Security](img/security/security-menu.png)

## Two-Factor Authentication

In order to enforce 2FA logins for your team members, you must have the 2FA

> option enabled in your organization's Account Settings.

Reviewer: ```suggestion
option enabled in your organization's account settings.
```

> 1. Click on the **account settings** link to be taken to the settings page

Reviewer: ```suggestion
1. Click on the **account settings** link to be taken to the settings page.
```

    ![Account Settings](img/security/account-settings.png)

> 1. Click the "Turn on Two-Factor Auth" button under the section titled

Reviewer: ```suggestion
1. Under the **Two-factor authentication** section, click **Turn on Two-Factor Auth**.
```

"Two-factor authentication",

    ![Turn on Two-Factor Auth](img/security/enable-2fa-button.png)}

> 1. Confirm your account password

Reviewer: ```suggestion
1. Confirm your account password.
```

    ![Password Dialog](img/security/password-dialog.png)

> 1. Follow the instructions on the screen to configure your 2FA device

Reviewer: ```suggestion
1. Follow the instructions on the screen to configure your 2FA device.
```

    ![Configure Device](img/security/configure-device.png)

> >Note:  If SAML authentication is configured for your team, Two-Factor Authentication will be bypassed. You can leave it enabled but you will not be prompted to enter a code when logging in.

Reviewer: ```suggestion
>Note: If SAML authentication is configured for your team, Two-Factor Authentication will be bypassed. You can leave it enabled but you will not be prompted to enter a code when logging in.
```

## Strong Passwords

> You can enforce strong passwords for all your team by requiring that all

Reviewer: ```suggestion
You can enforce strong passwords for all members of your team by requiring that all
```

passwords meet certain requirements for complexity.

> 1. Check the box next to "Require custom password complexity for team"

Reviewer: ```suggestion
1. Check the box next to **Require custom password complexity for team**.
```

    ![Password Complexity](img/security/password-complexity.png)

1. Provide a minimum password length, and required number of letters,
digits, and symbols

> 1. Click "Save Changes"

Reviewer: ```suggestion
1. Click **Save Changes**.
```
